programs/ssl/ssl_server2.c

```diff
@@ -1041,8 +1056,8 @@ int main( int argc, char *argv[] )
         }
         else if( strcmp( p, "renego_period" ) == 0 )
         {
-            opt.renego_period = atoi( q );
-            if( opt.renego_period < 2 || opt.renego_period > 255 )
+            if( sscanf( q, "%" SCNu64, &opt.renego_period ) != 1 ||
```

Comment:
It seems that the format compatibility macros were only introduced in C99. I was using these here because it is a straight-forward way to parse the command line value. I will try to find another way to achieve the same purpose.